server/schedule/operator/operator.go

```diff
@@ -228,10 +228,13 @@ func (o *Operator) CheckTimeout() bool {
 	if o.CheckSuccess() {
 		return false
 	}
-	if o.kind&OpRegion != 0 {
-		return o.status.CheckTimeout(SlowOperatorWaitTime)
+	currentStep := int(atomic.LoadInt32(&o.currentStep))
+	startTime := o.GetStartTime()
+	if 0 < currentStep && currentStep-1 < len(o.steps) {
+		startTime = time.Unix(0, atomic.LoadInt64(&(o.stepsTime[currentStep-1])))
 	}
-	return o.status.CheckTimeout(FastOperatorWaitTime)
+	step := o.steps[currentStep]
+	return o.status.CheckStepTimeout(startTime, step, o.ApproximateSize)
```

Comment:
Should we move them into `Check` function?

Reply:
`Check` will boost  `OpStep` and update `Operator`'s  status  that  depences on current lastest origin info.  But `checkTimeout` should no depence  it. 
maybe some same codes should be exract.